### PR TITLE
More "get rid of just-python invocation"

### PR DESCRIPTION
**Pull request by NattyNarwhal**
Split into two commits - the shebang stuff might be controversial if CentOS 6 or whatever lacks Python 3.

### CODE REVIEW

**Comment by kg** [comment]
Please do the shebang changes in a separate PR, because while the $(PYTHON) stuff looks good I'm not comfortable with the shebang changes

File: netcore/Makefile [hunk 1]
```diff
@@ -320,4 +320,4 @@ jitdiff:
 	cd tools/jitdiff && $(DOTNET) run -c Release -- "$(BEFORE)" "$(AFTER)"
 
 distdir:
-distclean:
\ No newline at end of file
+distclean:
```
Comment (kg) [inline netcore/Makefile]: Line ending change?

Comment (NattyNarwhal) [reply netcore/Makefile]: My editor probably did this, sorry.

**Comment by lambdageek** [comment]
@monojenkins build pkg

**Comment by lambdageek** [comment]
@monojenkins build OS X Extended

**Comment by lewurm** [comment]
@monojenkins build OS X Extended

**Pull request merged**